src/Simulator/FloodyFluidSimulator.cpp

```diff
@@ -111,9 +111,11 @@ void cFloodyFluidSimulator::SimulateBlock(cChunk * a_Chunk, int a_RelX, int a_Re
 		
 		// If source creation is on, check for it here:
 		if (
-			(m_NumNeighborsForSource > 0) &&  // Source creation is on
-			(MyMeta == m_Falloff) &&          // Only exactly one block away from a source (fast bail-out)
-			!IsPassableForFluid(Below) &&     // Only exactly 1 block deep
+			(m_NumNeighborsForSource > 0) &&    // Source creation is on
+			(MyMeta == m_Falloff) &&            // Only exactly one block away from a source (fast bail-out)
+				(!IsPassableForFluid(Below) ||  // Only exactly 1 block deep
```

Comment:
Seems legit. I first tried with adding a space before (Below == m_Stationary), but that failed in the style-checker.